lib/OpenQA/WebAPI/Controller/API/V1/Table.pm

```diff
@@ -320,4 +282,51 @@ sub destroy {
     $self->render(json => {result => int($ret)});
 }
 
+=over 4
+
+=item prepare_settings()
+
+Internal method to prepare settings when add or update admin table.
+Use by both B<create()> and B<update()> method.
+
+=back
+
+=cut
+
+sub prepare_settings {
+    my ($self, $table, $entry) = @_;
+    my $validation = $self->validation;
+
+    for my $par (@{$tables{$table}->{required}}) {
+        $validation->required($par);
+        if (!defined $self->param($par)) {
```

Comment:
If you're using validation you need to use `$validation->param()` instead of `$self->param()` consistently.

Reply:
I will fix it.